_data/docsarchive/nginx-docs.conf

```diff
@@ -0,0 +1,27 @@
+# prevent buffering to disk
+proxy_buffering off;
+
+# prevent writing large requests to disk
+# values should be equal
+client_body_buffer_size 5m;
+client_max_body_size    5m;
+
+server {
+  listen       80;
+  server_name  docs.docker.com;
+  resolver     127.0.0.11 ipv6=off valid=30s;
+  error_page   404 /404.html;
```

Comment:
Can you also redirect HTTP 403 to 404.html? Sometimes people try to change the url and land on directories that don't have index files. In that case they should see the same 404 page instead of a "permission denied error".